config/config.go

```diff
@@ -389,6 +389,24 @@ type GlobalConfig struct {
 	QueryLogFile string `yaml:"query_log_file,omitempty"`
 	// The labels to add to any timeseries that this Prometheus instance scrapes.
 	ExternalLabels labels.Labels `yaml:"external_labels,omitempty"`
+	// An uncompressed response body larger than this many bytes will cause the
+	// scrape to fail. 0 means no limit.
```

Comment:
I guess we need "0 means no limit." everywhere on those below?

Reply:
I've done it for every new limit field in the global config, and I've made sure to also make the scrape config limits consistent in this regard.